### PR TITLE
Use filename

### PR DESCRIPTION
**Pull request by antazoey**

### CODE REVIEW

**Review by kiran-chaudhary — APPROVED** [review]
LGTM

File: Packs/Code42/Integrations/Code42/Code42.yml [hunk 1]
```diff
@@ -419,11 +419,20 @@ script:
       description: The username of the High Risk Employee.
       type: Unknown
   - arguments:
-    - default: false
+    - auto: PREDEFINED
+      default: false
       description: To filter results by employees who have these risk tags. Space
```
Comment (alanag13) [inline Packs/Code42/Integrations/Code42/Code42.yml]: The description of all these fields still say they are space delimited.

File: Packs/Code42/Playbooks/playbook-Code42_Exfiltration_Playbook.yml [hunk 2]
```diff
@@ -867,21 +867,32 @@ tasks:
...
       playbookName: Code42 File Download
       type: playbook
       iscommand: false
       brand: ""
-      description: ""
     nexttasks:
       '#none#':
       - "5"
     scriptarguments:
+      Filename:
```
Comment (alanag13) [inline Packs/Code42/Playbooks/playbook-Code42_Exfiltration_Playbook.yml:886]: Why did this existing playbook need to be modified?

File: Packs/Code42/Playbooks/playbook-Code42_File_Download.yml [hunk 3]
```diff
@@ -21,6 +21,11 @@ inputs:
       root: File
       transformers:
       - operator: uniq
+- description: The name of the file to save as.
```
Comment (alanag13) [inline Packs/Code42/Playbooks/playbook-Code42_File_Download.yml:24]: Similar to some other conversations in slack, I'm questioning the value of having a playbook that does virtually the same thing as a command, but we can discuss more offline.

File: Packs/Code42/Playbooks/playbook-Code42_Exfiltration_Playbook.yml (hunk 2 above)
Comment (antazoey) [reply Packs/Code42/Playbooks/playbook-Code42_Exfiltration_Playbook.yml:886]: It now passes a parameter of the filename to the command for saving the file

File: Packs/Code42/Integrations/Code42/Code42.yml (hunk 1 above)
Comment (antazoey) [reply Packs/Code42/Integrations/Code42/Code42.yml]: fixed

**Pull request merged**